packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -27,7 +28,15 @@ const styles = {
 };
 
 const GlobalHeader = ({ editUrl, className, search }) => {
-  const [isOverlayOpen, setIsOverlayOpen] = useState(false);
+  const location = useLocation();
+  const query = new URLSearchParams(location.search);
+  const [isOverlayOpen, setIsOverlayOpen] = useState(query.has('q'));
+
+  useEffect(() => {
+    if (isOverlayOpen && !new URLSearchParams(location.search).has('q'))
+      navigate(location.pathname + '?q=');
+    if (!isOverlayOpen) navigate(location.pathname);
```

Comment:
```suggestion
    if (isOverlayOpen && !new URLSearchParams(location.search).has('q')) {
      navigate(location.pathname + '?q=');
    }
    
    if (!isOverlayOpen) {
      navigate(location.pathname);
    }
```

I'm finding a bit hard to parse the conditionals since at a glance it looks like the code is just indented incorrectly rather than being part of the `if` condition. Feel free to accept/ignore this suggestion, but it helps me read this a bit easier when parsing through the code.